hoomd/hpmc/field.py

```diff
@@ -32,11 +37,10 @@ def __init__(self):
         # nothing else to do.
 
 
-class lattice_field(_external):
+class LatticeField(_HOOMDBaseObject):
```

Comment:
This class will need to inherit from a different (probably new) class that all of the external fields inherit from.

Reply:
Done.